TYPO3-Solr/ext-solr
# [WIP][FEATURE] Use TYPO3 site configuration for solr connection info

# What this pr does

Add the option to configure your solr connection info through the TYPO3 site configuration instead of `typoscript` and/or `$GLOBALS['TYPO3_CONF_VARS']['EXTCONF']['solr']['sites']....`

# How to test

Configure site with the `Site management tool` and check if cores can be found

Fixes: #2238


# Todo
- Check if we can hide "initialise conneciton" when site config is used
- Add missing fields to site configuration
- Add proper tests
- Fix solr core -> path resolving
- Cleanup todo in code (remove/deprecate unused methods)
- Think about how we handle username and password (site configuration could be public accessable)
- Documentation of Legacy and TYPO3Managed site mode
